win32ss/w32ksvc32.h

```diff
@@ -694,3 +694,59 @@
 // For Wine DX
     SVC_(GdiDdDDICreateDCFromMemory, 1)
     SVC_(GdiDdDDIDestroyDCFromMemory, 1)
+// ReactOS Display Driver Model
+    SVC_(GdiDdDDICheckExclusiveOwnership, 0)
```

Comment:
We should really think of a smarter way to do this, maybe some template haxxory or a (python?) script.